tests/ha/fencing.pm

```diff
@@ -25,8 +25,15 @@ sub run {
     check_cluster_state;
     barrier_wait("CHECK_BEFORE_FENCING_END_$cluster_name");
 
-    # Fence the master node
-    assert_script_run 'crm -F node fence ' . get_node_to_join if is_node(2);
+    # Fence the master node with sysrq or crm node fence
+    # Sysrq fencing is more a real crash simulation
+    if (get_var('USE_SYSRQ_FENCING')) {
+        record_info('Fencing info', 'Fencing done by sysrq');
+        script_run 'echo b > /proc/sysrq-trigger' if (get_var('HA_CLUSTER_INIT'));
```

Comment:
You could get rid of this: http://1a102.qa.suse.de/tests/31#step/fencing/15

With `type_string` instead of `script_run`. If you change it, remember to add a `\n` after the command in `type_string`.